core/trino-main/src/main/java/io/trino/execution/buffer/OutputBufferInfo.java

```diff
@@ -33,6 +34,7 @@
     private final long totalRowsSent;
     private final long totalPagesSent;
     private final List<BufferInfo> buffers;
+    private final TDigestHistogram utilization;
```

Comment:
Would it be possible to propagate this into `io.trino.operator.TaskStats` instead?

Could you also aggregate utilization histogram in `StageStats`?

Reply:
Yes, it could be part of `OperatorStats`. It is even better approach. Thanks!